docs/source/status.rst

```diff
@@ -331,7 +331,7 @@ Third-Party Data Formats
 +=============================+=========+=========+=======+============+=======+=======+=======+
 | Avro                        |         | R       |       |            |       |       |       |
 +-----------------------------+---------+---------+-------+------------+-------+-------+-------+
-| CSV                         | R/W     |         | R/W   |            |       | R/W   | R/W   |
+| CSV                         | R/W     | R       | R/W   |            |       | R/W   | R/W   |
```

Comment:
We should add the caveat that it's through JNI (like for ORC below).

Reply:
Hello @lidavidm Thank you for review! Done